### ISSUE TITLE
`copyEventFromReader()` Processing Instruction event bug

### ISSUE THREAD

**Issue by dan2097** [FasterXML/woodstox#132]
Tested with v6.2.6. The code below outputs :
`<description<?pi ?>>foo</description>`
which I believe is malformed as the processing instruction event is within the start element.

```
String in = "<description><?pi?>foo</description>";
XMLStreamReader2 reader = (XMLStreamReader2) new WstxInputFactory().createXMLStreamReader(new StringReader(in));
ByteArrayOutputStream baos = new ByteArrayOutputStream();
XMLStreamWriter2 writer = (XMLStreamWriter2) new WstxOutputFactory().createXMLStreamWriter(baos, "UTF-8");
while (reader.hasNext()) {
  reader.next();
  writer.copyEventFromReader(reader, true);//true or false give same result
}
writer.close();
System.out.println(baos.toString("UTF-8"));
```

**Comment by cowtowncoder** [comment]
Ok; took a while to find time, and then a bit more to isolate the problem but I found it & fix is checked in now.
Will see if there is anything else before releasing 6.2.7

**Comment by cowtowncoder** [comment]
Decided time was ripe, so just released version 6.2.7. Should be up at Maven Central soon.

Also filed new issues for dependency upgrade for OSGi.core & JDK baseline upgrade to Java 8 -- so considering 6.3.0 release in near future -- with only dependency upgrades.
